bindings/nodejs/examples/how_tos/sign_evm/sign_evm.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    CoinType,
+    HD_WALLET_TYPE,
+    HARDEN_MASK,
+    initLogger,
+    SecretManager,
+    utf8ToHex,
+} from '@iota/sdk';
+require('dotenv').config({ path: '.env' });
+
+// In this example we will sign with secp256k1.
+// Run with command:
+// yarn run-example ./how_tos/sign_evm/sign_evm.ts
+
+const FOUNDRY_METADATA = {
+    standard: 'IRC30',
+    name: 'NativeToken',
+    description: 'A native token',
+    symbol: 'NT',
+    decimals: 6,
+    logoUrl: 'https://my.website/nativeToken.png',
+};
```

Comment:
I think this should be some other "random" message here, signing the foundry metadata is interesting with ed25519 because it's needed for the token registry https://github.com/iotaledger/token-registry, but an EVM address can't be used for that

Reply:
Does it really matter though?